playwright/test.ts

```diff
@@ -0,0 +1,202 @@
+import { PNG } from "pngjs";
```

Comment:
Aggiungerei un po' di logs nel runtime dell'applicazione per poter facilmente debuggare eventuali problemi. Mi era tornato molto utile in Bocelli. 

Potresti pensare di avere due livelli di log `defualt` e `verbose`. 

Il livello `default` è praticamente quello attuale (vedi tu se ci sono alcune cose dei punti successivi che possono essere considerati `default`). 

Il livello `verbose` dovrebbe dirci quando: 
- il browser carica una pagina con un certo device name (o con un certo viewport in caso di desktop device)
- la pagina si è caricata
- lo screenshot è stato eseguito (e con che nome viene salvato)
- lo screenshot matcha la baseline o meno (in caso di esecuzione di test)
- la pagina è stata chiusa 
- il test è passato o meno